crates/codegen/test_files/module_analysis/bidirectional.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/codegen/tests/module_analysis.rs
+input_file: test_files/module_analysis/bidirectional.sntn
+---
+ModuleDependency: Bidirectional
+
+SCC: [`%fx`]
+SCC: [`%f1`, `%f2`]
+SCC: [`%f3`]
+SCC: [`%f4`]
+SCC: [`%f5`]
+SCC: [`%f6`]
+SCC: [`%f7`]
+
+`%fx` = FuncInfo { is_non_recursive: false, flow: OutgoingOnly, is_leaf: true }
```

Comment:
The FuncInfo for externally-defined `%fx` is weird; local functions call it, so its flow is known to be Incoming, and we should assume that it may be Bidirectional. We also can't say that it's a leaf.

Reply:
Yeah, this is bad. I initially thought that we should remove the external function from the result, but also hesitated to do that because it breaks the consistency between the result and `FuncStore` or other modules, which always manage the mapping from each `FuncRef` to some information.

But, I'm inclined to remove that from the `ModuleInfo` because the information probably won't be used by optimization/codegen anyways. What's your thought?

Reply:
Yeah, I think it's fine to remove it and for the backend to assume the most conservative for calls to mysterious external functions.